hr_timesheet_activity_begin_end/views/hr_analytic_timesheet.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="hr_timesheet_line_tree" model="ir.ui.view">
```

Comment:
optional change:
use original id (hr_timesheet_line_tree)

> prefer naming the inherited view after the original over using a name which follows the guidelines. This eases looking up the original view and other inheritance if they all have the same name.

Reply:
I think the original id is being used, even before migrating it was the same. Thanks anyway. :)

Reply:
must have read some other line, while checking it...